src/components/PhoneNumberInput/index.js

```diff
@@ -36,6 +46,7 @@ class PhoneNumberInput extends Component {
         country={this.state.country}
         inputClassName={`${style.mobileInput}`}
         className={`${style.phoneNumberContainer}`}
+        flagComponent={ FlagComponent }
```

Comment:
Hi @elhesuu, nice to meet you.
Looks good. I didn't know the library had this prop. 👍

Reply:
Nice to meet you too @stefaniacardenas !